starfile/writer.py

```diff
@@ -19,7 +19,7 @@
 class StarWriter:
     def __init__(self, dataframes: Union[pd.DataFrame, Dict[pd.DataFrame], List[pd.DataFrame]],
                  filename: PathLike, overwrite: bool = False, float_format: str = '%.6f',
-                 sep: str = '\t', na_rep: str = '<NA>', force_loop: bool = False):
+                 sep: str = '\t', na_rep: str = '<NA>', force_loop: bool = True):
```

Comment:
```suggestion
                 sep: str = '\t', na_rep: str = '<NA>', force_loop: bool = False):
```

reverting this to fix test failure

Reply:
I will go ahead with your preferred fix.

But just for my understanding: what is the difference between these two files `starfile/functions.py` and `starfile/writer.py`? And does it matter that they will have different defaults for `force_loop`?

Reply:
The function is the standard/easy/expected way of writing files, provides a simple interface that people tend to be more comfortable using. The class in `writer.py` does the legwork of actually writing the file, classes are useful for this kind of thing because they allow you to easily keep track of state within the object. 

Changing the defaults makes the most sense on the function because that's what most people will use :)